packages/ses/src/make-evaluate-factory.js

```diff
@@ -1,5 +1,10 @@
 import { arrayJoin } from './commons.js';
 
+// The original unsafe untamed Function constructor, which must not escape.
+// Sample at module initialization time, which is before lockdown can
+// repair it. Use it only to build powerless abstractions.
+const FERAL_FUNCTION = Function;
```

Comment:
👍 Clearly the opposite of tame. Implies existence of “dread function” and “dire function”.